src/components/data-table/DataTable.vue

```diff
@@ -115,11 +120,11 @@ export default {
     props: {
         columns: {
             type: Array,
-            default: null
+            default: () => []
```

Comment:
I also considered the default being undefined (i.e. rows and columns _must_ be set), but that's potentially not backwards compatible even though it captures the intended use better.